### PR TITLE
DACT-671: TM01 Format Corp Director Name to Uppercase

### PR DESCRIPTION
**Pull request by leighmcclurg**
https://companieshouse.atlassian.net/browse/DACT-671

### CODE REVIEW

File: src/controllers/remove.director.check.answers.controller.ts [hunk 1]
```diff
@@ -50,17 +50,22 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
       }
     }
 
+    //Check if corporate director and format name based on corporate or individual director
     var corporateDirector = false;
+    var directorName = "";
     if(equalsIgnoreCase(companyOfficer.officerRole, OFFICER_ROLE.CORPORATE_DIRECTOR) || equalsIgnoreCase(companyOfficer.officerRole, OFFICER_ROLE.CORPORATE_NOMINEE_DIRECTOR)){
       corporateDirector = true;
-    }
+      directorName = (companyOfficer.name).toUpperCase();
```
Comment (AndreKramer7654) [inline src/controllers/remove.director.check.answers.controller.ts:58]: no need for ()

File: src/controllers/remove.director.check.answers.controller.ts [hunk 2]
```diff
@@ -50,17 +50,22 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
       }
     }
 
+    //Check if corporate director and format name based on corporate or individual director
     var corporateDirector = false;
```
Comment (AndreKramer7654) [inline src/controllers/remove.director.check.answers.controller.ts:54]: var is consistent with usage here but I'd say let is preferred

File: src/controllers/remove.director.controller.ts [hunk 3]
```diff
@@ -44,13 +45,21 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     // Get the director name from company appointments
     const appointment: CompanyAppointment = await getCompanyAppointmentFullRecord(session, companyNumber, appointmentId);
 
+    //Format name based on corporate or individual director
```
Comment (AndreKramer7654) [inline src/controllers/remove.director.controller.ts:48]: could this be a shared method - seems same as above.

File: src/controllers/remove.director.check.answers.controller.ts [hunk 4]
```diff
@@ -50,17 +50,22 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
...
+    let directorName = "";
```
Comment (AndreKramer7654) [inline src/controllers/remove.director.check.answers.controller.ts:55]: can we use  formatDirectorNameForDisplay() here too?

File: src/controllers/remove.director.controller.ts [hunk 5]
```diff
@@ -147,7 +154,20 @@ function displayErrorMessage(validationErrors: ValidationError[], appointment: C
   });
 }
 
-function displayPopulatedPage(dateFields: string[], appointment: CompanyAppointment, req: Request, res: Response<any, Record<string, any>>) {
+/**
+ * Format the directors name based on the role from Company Appointment
+ */
+function formatDirectorNameForDisplay (appointment: CompanyAppointment): string {
+ let directorName = "";
```
Comment (AndreKramer7654) [inline src/controllers/remove.director.controller.ts:161]: indent

File: src/controllers/remove.director.check.answers.controller.ts (hunk 4 above)
Comment (leighmcclurg) [reply src/controllers/remove.director.check.answers.controller.ts]: I have moved the formatDirectorNameForDisplay() method into utils.format, and it's now being used in both remove.director.controller and remove.director.check.answers.controller

**Review by AndreKramer7654 — APPROVED** [review]

**Review by PhilipGlover — APPROVED** [review]
👍

**Pull request merged**